pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA             = "n/a"
 	name        string = "cert-operator"
 	source      string = "https://github.com/giantswarm/cert-operator"
-	version            = "0.1.0"
+	version            = "0.3.0"
```

Comment:
Version?

Reply:
1.0.0 makes sense to me even though it's not that big of a change.